pom.xml

```diff
@@ -140,8 +140,21 @@
                 <version>1.12</version>
                 <executions>
                     <execution>
-                        <id>add-test-source</id>
+                        <id>add-groovy-sources</id>
                         <phase>generate-sources</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>src</source>
```

Comment:
no need for adding `src` twice. `src` has already been set with the the `sourceDirectory` tag inside the `build` section. --> IMO this line can be removed.

Reply:
removed `sourceDirectory` entry